src/status_im/contexts/profile/edit/name/utils.cljs

```diff
@@ -0,0 +1,43 @@
+(ns status-im.contexts.profile.edit.name.utils
+  (:require [clojure.string :as string]
+            [utils.i18n :as i18n]))
+
+(def min-length 5)
+
+(def max-length 24)
+
+(def emoji-regex
+  (new
+   js/RegExp
+   #"(\u00a9|\u00ae|[\u2000-\u3300]|\ud83c[\ud000-\udfff]|\ud83d[\ud000-\udfff]|\ud83e[\ud000-\udfff])"
+   "i"))
+
+(defn has-emojis [s] (re-find emoji-regex s))
+
+(def common-names ["Ethereum" "Bitcoin"])
+
+(defn has-common-names [s] (pos? (count (filter #(string/includes? s %) common-names))))
+
+(def status-regex (new js/RegExp #"^[a-zA-Z0-9\-_ ]+$"))
+
+(defn has-special-characters [s] (not (re-find status-regex s)))
+
+(defn name-too-short [s] (< (count (string/trim (str s))) min-length))
+
+(defn name-too-long [s] (> (count (string/trim (str s))) max-length))
+
+(defn validation-name
+  [s]
+  (cond
+    (or (= s nil) (= s ""))      nil
```

Comment:
this is the same validation as create profile page??
We should abstract that logic to a common place 👍